tokens/properties/mixins/InlineProgressBar.json

```diff
@@ -0,0 +1,39 @@
+{
+    "wikit-InlineProgressBar": {
+        "background-color": {
+            "value": "{background.color.progressive.default.value}"
+        },
+        "height": {
+            "value": "{dimension.static.size.50.value}"
+        },
+        "animation": {
+            "duration": {
+                "value": "{animation.loader.duration.slow.value}"
+            }
+        },
+        "track": {
```

Comment:
The "track" is the wrapper that contains both inline progress bar typologies (determinate and indeterminate). The width of the track is 100% by default, and therefore is ultimately determined by its own contextual wrapper. 
Note: It might make more sense to let the progress bars inherit the height of the track instead of creating two tokens with the same value. Suggestions welcome.